downToMeet/downToMeet
# Add authentication support

No actual OAuth login support yet. This merely allows the server to recognize authenticated clients.

Needs #10.

@jamieliu386 @conniec218 @Timthetic please take a look.

----

Testing:

1. `make start`
1. Visit http://localhost:9000/restricted. You should see an error.
1. Visit http://localhost:9000/set-cookie?user_id=tim. You should see "setting user ID to tim".
1. Visit http://localhost:9000/restricted again. You should now see "user ID is tim".

(When visiting the above links, make sure to right-click and then select "open in new tab". Otherwise things may not work properly.)
